<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1021</width>
    <height>630</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideNone</enum>
      </property>
      <widget class="QWidget" name="tab_1">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>778</width>
         <height>498</height>
        </size>
       </property>
       <attribute name="title">
        <string>Analizar Indicador</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <layout class="QVBoxLayout" name="vlay_search">
            <item>
             <layout class="QHBoxLayout" name="hlay_search1">
              <item>
               <widget class="QLabel" name="lb_indicador">
                <property name="text">
                 <string>   Indicador: </string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="cb_indicadores">
                <property name="maximumSize">
                 <size>
                  <width>900</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="maxVisibleItems">
                 <number>20</number>
                </property>
                <property name="sizeAdjustPolicy">
                 <enum>QComboBox::AdjustToContents</enum>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="hlay_search2">
              <item>
               <widget class="QLabel" name="lb_busqueda">
                <property name="text">
                 <string>   Búsqueda: </string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="cb_searchtype"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="hlay_search3">
              <item>
               <widget class="QLabel" name="lb_desde">
                <property name="text">
                 <string>   Desde</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDateEdit" name="de_inicio">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="dateTime">
                 <datetime>
                  <hour>0</hour>
                  <minute>0</minute>
                  <second>0</second>
                  <year>2018</year>
                  <month>1</month>
                  <day>1</day>
                 </datetime>
                </property>
                <property name="currentSection">
                 <enum>QDateTimeEdit::DaySection</enum>
                </property>
                <property name="calendarPopup">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Hasta</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDateEdit" name="de_final">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="dateTime">
                 <datetime>
                  <hour>0</hour>
                  <minute>0</minute>
                  <second>0</second>
                  <year>2018</year>
                  <month>1</month>
                  <day>1</day>
                 </datetime>
                </property>
                <property name="calendarPopup">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="hlay_search4">
              <item>
               <widget class="QPushButton" name="bt_limpiar">
                <property name="text">
                 <string>Limpiar</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="bt_search">
                <property name="text">
                 <string>Buscar</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QListWidget" name="lw_resultados">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>600</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>778</width>
         <height>498</height>
        </size>
       </property>
       <attribute name="title">
        <string>Desempeño de personal</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <layout class="QVBoxLayout" name="vlay_search_2">
              <item>
               <layout class="QHBoxLayout" name="hlay_search1_2">
                <item>
                 <widget class="QLabel" name="lb_operador_text">
                  <property name="text">
                   <string>Operador:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="cb_operadores">
                  <property name="maximumSize">
                   <size>
                    <width>900</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="maxVisibleItems">
                   <number>20</number>
                  </property>
                  <property name="sizeAdjustPolicy">
                   <enum>QComboBox::AdjustToContents</enum>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="hlay_search2_2">
                <item>
                 <widget class="QLabel" name="lb_busqueda_op">
                  <property name="text">
                   <string>   Búsqueda: </string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="cb_searchtype_op"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="hlay_search3_2">
                <item>
                 <widget class="QLabel" name="lb_desde_op">
                  <property name="text">
                   <string>   Desde</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDateEdit" name="de_inicio_op">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="dateTime">
                   <datetime>
                    <hour>0</hour>
                    <minute>0</minute>
                    <second>0</second>
                    <year>2018</year>
                    <month>1</month>
                    <day>1</day>
                   </datetime>
                  </property>
                  <property name="calendarPopup">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="lb_hasta_op">
                  <property name="text">
                   <string>Hasta</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDateEdit" name="de_final_op">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="dateTime">
                   <datetime>
                    <hour>0</hour>
                    <minute>0</minute>
                    <second>0</second>
                    <year>2018</year>
                    <month>1</month>
                    <day>1</day>
                   </datetime>
                  </property>
                  <property name="calendarPopup">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="hlay_search4_2">
                <item>
                 <widget class="QPushButton" name="bt_limpiar_op">
                  <property name="text">
                   <string>Limpiar</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="bt_search_op_perform">
                  <property name="text">
                   <string>Buscar</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_4"/>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_5">
                <item>
                 <widget class="QLabel" name="lb_op_detalles">
                  <property name="font">
                   <font>
                    <weight>75</weight>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="text">
                   <string>Detalles</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="lb_indicator_name">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="text">
                   <string>Indicador</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QListWidget" name="lw_resultados_op_detalles"/>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <property name="sizeConstraint">
             <enum>QLayout::SetMinimumSize</enum>
            </property>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <property name="sizeConstraint">
               <enum>QLayout::SetMinimumSize</enum>
              </property>
              <item>
               <widget class="QLabel" name="label_2">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>10</verstretch>
                 </sizepolicy>
                </property>
                <property name="font">
                 <font>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>Resultados Bundle para </string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="lb_current_operator">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>Operador</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QListWidget" name="lw_resultados_op">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>600</width>
                <height>0</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Exportar</string>
       </attribute>
       <widget class="QRadioButton" name="radioButton">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>130</y>
          <width>119</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>RadioButton</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1021</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
